src/test/java/spoon/support/visitor/java/JavaReflectionTreeBuilderTest.java

```diff
@@ -80,10 +80,10 @@ public void testScannerClass() {
 		assertEquals("java.lang.Class", aClass.getQualifiedName());
 		//The Class extends Object, but CtElementImpl (made from sources) getSuperclass() returns null. See CtTypeInformation#getSuperclass() comment.
 		assertNull(aClass.getSuperclass());
-		assertTrue(aClass.getSuperInterfaces().size() > 0);
-		assertTrue(aClass.getFields().size() > 0);
-		assertTrue(aClass.getMethods().size() > 0);
-		assertTrue(aClass.getNestedTypes().size() > 0);
+		assertTrue(!aClass.getSuperInterfaces().isEmpty());
```

Comment:
I found it hard to read this way: could you refactor all those `assertTrue(!...)` to `assertFalse(...)` instead?